src/main/java/com/mraof/minestuck/alchemy/GristHelper.java

```diff
@@ -139,24 +146,56 @@ public static void decrease(Level level, PlayerIdentifier player, GristSet set)
 	public static void decreaseAndNotify(Level level, PlayerIdentifier player, GristSet set, GristHelper.EnumSource source)
 	{
 		decrease(level, player, set);
-		notify(level.getServer(), player, set, source, false);
+		GristSet total = PlayerSavedData.getData(player, level).getGristCache();
```

Comment:
From my understanding, the grist cache would already be synced to `ClientPlayerData` on the call of `decrease()` (or `increase()`), so it wouldn't need to be sent with the `GristToastPacket` as well.
However, if that change were to be made, the packet would instead need to know whether the toast was for the player themselves, or their editmode target.

Reply:
oh so would i just be able to get the total within the grist toast class?

Reply:
Yeah